### DISCUSSION TITLE
Release plan

### DISCUSSION THREAD

**Discussion by Nezisi** [AsyncHttpClient/async-http-client#1897] [Q&A] [answered]
Important thing first: Really glad you keep the project alive. Thanks alot for your efforts!

As play framework 2.9 seems to draw close, I wanted to know if the AHC release 3 is getting close, too?

Main project which uses the AHC library is the play-ws-client, which uses shading for the AHC library and the netty library AHC depends upon.

**Comment by hyperxpro** [comment, marked as answer]
I am expecting it to be Q3 2023.

**Reply by AshwinPrabhuB** [reply]
Is this plan on schedule Aayush?

**Reply by hyperxpro** [reply]
A little delay, just finalizing APIs.
